libr/include/r_util/r_strbuf.h

```diff
@@ -7,9 +7,10 @@ extern "C" {
 
 typedef struct {
 	char buf[32];
-	int len;
+	size_t len;
 	char *ptr;
-	int ptrlen;
+	size_t ptrlen;
+	bool ro;
```

Comment:
Maybe add a comment here what ro means (weak vs. strong reference).

Reply:
maybe better to call it `weak_ref` and avoid the comment

Reply:
ok too

Reply:
done